test/conversion.test.js

```diff
@@ -51,6 +97,17 @@ describe('Curl converter should', function() {
 		done();
 	});
 
+	it('throw an error if POST body option is given with HEAD without --get/-G', function(done) {
+		convert({
+			type: 'string',
+			data: 'curl -I http://example.com -d "a=b"'
+		}, function (err, result) {
+			expect(result.result).to.equal(false);
+			expect(result.reason).to.equal('Error while parsing cURL: You can only select one HTTP request method!');
```

Comment:
@Dhroov7 No exclamation marks needed. Also, we can specify the exact error too - `Both --head and --data are not supported`